exporters/pelorus/timeutil.py

```diff
@@ -62,6 +62,26 @@ def parse_guessing_timezone_DYNAMIC(timestring: str, format: str) -> datetime:
         return parsed.replace(tzinfo=timezone.utc)
 
 
+def to_epoch_from_string(
+    timestring: str, allowed_length: list[int] = [10, 13]
+) -> datetime:
+    """
+    It's really EPOCH to EPOCH conversion with datetime return object.
+
+    If timestring matches expected EPOCH format a proper datetime object is returned,
+    otherwise raises ValueError and consumer of this function probably want's to use
+    other format of timestamp.
```

Comment:
Since this is just a best guess and they'll want to fall back, maybe this should just return `None` if it doesn't work? Then you could do `(to_epoch_from_string(ts) or parse_guessing_timezone_DYNAMIC(ts, format)).timestamp()`

Reply:
I prefer to give explicit error rather then silently fall with None in case someone will want to use that function later, but of course can change if you disagree.

Reply:
I think this depends on the length discussion-- I'll make a bigger followup comment.